src/Orchard.Cms.Web/Modules/Orchard.Title/Views/TitlePart.Edit.cshtml

```diff
@@ -5,4 +5,19 @@
     <label asp-for="Title">@T["Title"] <span asp-validation-for="Title"></span></label>
     <input asp-for="Title" class="form-control" />
     <span class="hint">@T["The title of the content item."]</span>
-</fieldset>
\ No newline at end of file
+</fieldset>
+
+<script at="Foot">
+    $(function () {
+        $('#@Html.IdFor(m => m.Title)')
+        .on('input', function () {
+            $(document).trigger('contentpreview:render');
+        })
+        .on('keyup', function (event) {
+            // handle backspace
+            if (event.keyCode == 46) {
+                $(document).trigger('contentpreview:render');
+            }
+        });
```

Comment:
I remember using this for my Operation Transform stuff in O1 and it doesnt work everywhere... or has issues. 

Could we use this? http://www.zurb.com/playground/jquery-text-change-custom-event